frontend/integration-tests/views/login.view.ts

```diff
@@ -2,6 +2,6 @@
 
 import { $ } from 'protractor';
 
-export const nameInput = $('input[name=login]');
+export const nameInput = $('input[name=username]');
```

Comment:
I'm wondering when was this changed. @gouyang , any idea, please?

Reply:
This is changed in OCP 3.11, you can check it at the ocp 3.11 login page. However, this part is fixing in openshift/console just a few days ago, so it would be okay to use `export const nameInput = $('#inputUsername');`.

Reply:
I've merged upstream to the web-ui yesterday. It seems to be there.

So closing here - feel free to reopen if you think otherwise.